src/cache.js

```diff
@@ -2,18 +2,18 @@
 
 var _cache = [];
 
-function EventHandler(token, eventName, handler, enabled) {
+function EventHandler(token, eventName, splitEventName, handler, enabled) {
```

Comment:
Only problem is in the `on()` function I need to also split the descriptor. So i'll be doing it multiple times